pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/dao/impl/PatientRecordDAOImpl.java

```diff
@@ -84,33 +190,73 @@ public void delete(Long id) {
 		entityManager.flush();
 	}
 
-	
+
 	@Override
 	public PatientRecordDTO getById(Long id) {
-		
+
 		PatientRecordDTO dto = null;
 		PatientRecordEntity pe = this.getEntityById(id);
-		
+
 		if (pe != null){
 			dto = new PatientRecordDTO(pe); 
 		}
 		return dto;
 	}
+	
+	@Override
+	public PatientOrganizationMapDTO getPatientOrgMapById(Long id) {
```

Comment:
Same comment about moving getPatientOrgMapById. The patientOrgMaps (maybe a better name for this would help - open to suggestions) are related to a Patient that we created in the pulse system and not so much related to the PatientRecords that are coming back from the organizations.